script/app-mlperf-inference-mlcommons-python/customize.py

```diff
@@ -261,6 +262,8 @@ def get_run_cmd_reference(
                 cmd = "./run_local.sh " + env['MLC_MLPERF_BACKEND'] + ' ' + \
                     env['MLC_MODEL'] + ' ' + env['MLC_MLPERF_DEVICE'] + " --scenario " + env['MLC_MLPERF_LOADGEN_SCENARIO'] + " " + env['MLC_MLPERF_LOADGEN_EXTRA_OPTIONS'] + \
                     scenario_extra_options + mode_extra_options + dataset_options
+                if env['MLC_MODEL'] in ["resnet50"]:
+                    cmd += f" --max-batchsize {env.get('MLC_MLPERF_LOADGEN_MAX_BATCHSIZE', '128')} "
```

Comment:
in line num 66 we are supporting batchsize in `MLC_MLPERF_LOADGEN_EXTRA_OPTIONS`. . So, may be we should add this as a default before that line.